### PR TITLE
product page

### PR DESCRIPTION
**Pull request by prath-dev**
Screenshot from 2022-02-21 23-54-11
Screenshot from 2022-02-21 23-57-19

### CODE REVIEW

**Comment by prath-dev** [comment]
.

**Comment by sah-shah7** [comment]
looks good, can be merged

File: Components/Css/products.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+    .sidebar {
+        background-color: white;
+        border-right: 2px solid darkblue;
```
Comment (som749898) [inline Components/Css/products.css:3]: use css variable for color

**Review by som749898 — COMMENTED** [review]
You can use css variables for color scheme, rest looks good to me

**Review by sumeetyeola — APPROVED** [review]

**Comment by prath-dev** [comment]
@som749898 @sah-shah7  thansk for your suggestion i will do changes in second version of my ecommerce .

**Pull request merged**